### DISCUSSION TITLE
You can lead 🐎 to 💧

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#851] [Galaxy Trivia] [answered]
But you can't make it what?

**Comment by peterbaccaro** [comment, marked as answer]
The phrase "**You can lead a horse to water, but you can't make it drink**" is a well-known idiom. 
It means that you can provide someone with an opportunity or the necessary means to do something, but you cannot force them to take advantage of it or to act in a certain way. 
The idiom is often used to convey the idea that while you can offer guidance or assistance, ultimately, individuals have their own free will and can choose whether or not to take action or accept the help offered to them.
